feeder/api/routers/pet.py

```diff
@@ -101,13 +105,13 @@ async def new_feed_event(pet_id: int, updated_event: ScheduledFeed):
     pet = await get_pet(pet_id)
 
     if not pet.device_hid:
-        return HTTPException(
+        raise HTTPException(
```

Comment:
Yah, so fun fact: All of these errors were improperly raised. I guess these test things really do help.

Reply:
Epic winning.